src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -77,12 +74,12 @@ void register_age_notOk() {
 
     @Test
     void register_passLength_ok() {
-        user.setPassword(CORRECT_PASSWORD);
+        user.setPassword(SHORT_PASSWORD);
         assertEquals(user, registrationService.register(user));
     }

```

Comment:
what is the difference between this test and `register_nonExistsUser_ok`?
if we have any difference, can we combine these methods?